src/script_lua.c

```diff
@@ -56,10 +56,10 @@ static char *libraries_allow_list[] = {
 };
 
 /* Lua API globals */
-static char *redis_api_allow_list[] = {
+static char *server_api_allow_list[] = {
     SERVER_API_NAME,
     REDIS_API_NAME,
-    "__redis__err__handler", /* error handler for eval, currently located on globals.
+    "__server__err__handler", /* error handler for eval, currently located on globals.
```

Comment:
I'm not clear if this is a breaking change or not. I was fine just leaving it as __redis__ for now. Since we want to backport this, I would like this PR to represent what we backport.

Reply:
From what I understand, this is only used to handle errors internally. It is not to be invoked by users anywhere. I changed it in both places, where it is registered and where it gets invoked from.

Reply:
People have complained about us touching globals in the past, even ones they weren't supposed to be using. I'm not sure if anyone would materially call this, but I would rather lean on the conservative side and not touch it.

Reply:
I will alias them. If I can't then I will duplicate them. 

@madolson Tables are definitely aliased because lua uses a reference for them. That's how, all the fields added after luaRegisterServerAPI to 'server' table magically appear under 'redis' table as well.